DQMOffline/L1Trigger/interface/L1TEfficiencyHarvesting.h

```diff
@@ -0,0 +1,108 @@
+#ifndef DQMOFFLINE_L1TRIGGER_L1TEFFICIENCYHARVESTING_H
+#define DQMOFFLINE_L1TRIGGER_L1TEFFICIENCYHARVESTING_H
+
+/**
+ * \file L1TEfficiencyHarvesting.h
+ *
+ * \author J. Pela, C. Battilana
+ *
+ */
+
+// system include files
+#include <memory>
+#include <unistd.h>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/EDAnalyzer.h"
+#include "FWCore/Framework/interface/ESHandle.h"
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/LuminosityBlock.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
+
+#include "FWCore/ParameterSet/interface/ParameterSet.h"
+
+#include "DQMServices/Core/interface/DQMStore.h"
+#include "DQMServices/Core/interface/MonitorElement.h"
+#include "FWCore/ServiceRegistry/interface/Service.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+#include "DQMServices/Core/interface/DQMEDHarvester.h"
+
+#include <TString.h>
```

Comment:
Hi @kreczko - this include should not be needed (nor iostream and fstream I think)